06-airflow-dag-example/README.md

```diff
@@ -1,15 +1,26 @@
-## lakeFS + Airflow Example
+## lakeFS + Github + Airflow - example
 
-### Run
+* Build a custom **CD process** to make your code and git metadata accessible for an Airflow DAG.
+* Use Airflow to orchestrate your ETL run.
+* Use [lakeFS Airflow provider](https://github.com/treeverse/airflow-provider-lakeFS) to:
+   * Version control your raw, intermediate and processed data
+   * Link code versions and the data generated by running them
+   * Link Airflow DAG code versions to the data generated during a DAG run
```

Comment:
Is this a prerequisites section, a description of what's to come, or simply instructions? The relation between these bullets and the next section is not clear.

I think it should be something like:
```suggestion
This sample will give you an idea of how you can use  the [lakeFS Airflow provider](https://github.com/treeverse/airflow-provider-lakeFS) to:
   * Version control your raw, intermediate and processed data.
   * Link between code versions and the data generated by running them.
```

The first two bullets are more like prerequisites or assumptions I guess?

Reply:
ok that's a nice way to think about it! changed it